assets/javascripts/discourse/lib/babble.js.es6

```diff
@@ -127,6 +127,18 @@ export default Ember.Object.create({
     return result
   },
 
+  highlightCount() {
+    if (!this.get('currentTopic.postStream')) { return }
+    const username = Discourse.User.current().username.toLowerCase();
+    // Interestingly, link_is_a_mention? method does something really similar to this.
+    // This is O(N), but this should be fast enough in practice.
+    return this.get('currentTopic.postStream.posts')
```

Comment:
Hrm, interesting.

This seems slow and a little fragile to me, but I'm willing to put it in to test it out. Seems like we could do an initial calculation (perhaps server-side), and then determine when new mentions come in via `handleNewPost`

One thing is that we'll want to use the new String.includes method for checking for mentions:

```
cooked.includes(`<a class="mention" href="/users/${username}"`)
```